### DISCUSSION TITLE
issue: Typescript error for Date or null type in defaultValues

### DISCUSSION THREAD

**Discussion by wh-lob** [react-hook-form/react-hook-form#11132] [Q&A]
### Version Number

7.46.0

### Codesandbox/Expo snack

https://codesandbox.io/s/late-sunset-hlzgmz?file=/src/App.tsx

### Steps to reproduce

1. Go to the codesandbox link
2. I need the start date to start as empty and the validation requires the user to enter a date.
3. Validation works as expected where an error appears if no date is entered.
4. See the defaultValues where `startDate` is flagged that it cannot be null

### Expected behaviour

The startDate defaultValue to be null with no typescript error.

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by supersnager** [comment]
The same situation with 7.51.0 using Yup

**Comment by lebadapetru** [comment]
is there a solution for this issue? I get ts errors for null values on defaultValues
